imdbsol.txt

```diff
@@ -1,7 +1,46 @@
 #!/bin/bash
 
-t=`cat imdb.txt`
-rating="${t#*imdbRating}"
-rating="${rating%%%%}"
-res=${rating:2:3}	
-echo $res
+mkdir tmp1
+cd tmp1/
+touch movielist.txt
+touch output.txt
+cd ..
+
+ls $1 > tmp1/movielist.txt
+
+for ((i=1;; i++)); do
+	
+	read name || break;
```

Comment:
Avoid using tabs for indentation. Use spaces instead. Go through the bash style guide on indentation: https://google.github.io/styleguide/shell.xml#Indentation